go/chat/storage/storage.go

```diff
@@ -313,7 +313,13 @@ func (h *HoleyResultCollector) Holes() int {
 	return h.holes
 }
 
-func (s *Storage) MaybeNuke(ctx context.Context, force bool, err Error, convID chat1.ConversationID,
+func (s *Storage) MaybeNukeLocked(ctx context.Context, force bool, err Error, convID chat1.ConversationID, uid gregor1.UID) Error {
```

Comment:
I don't like this name, usually you put `Locked` on the end if you already have the lock. I would make the lower case name have this.

Reply:
sgtm!